apps/client/src/features/editor/page-editor.tsx

```diff
@@ -161,6 +161,10 @@ export default function PageEditor({
         scrollMargin: 80,
         handleDOMEvents: {
           keydown: (_view, event) => {
+            if ((event.ctrlKey || event.metaKey) && event.key === 's') {
```

Comment:
Hi! Your solution will work, but it is not very reliable. When switching keyboard layouts, this function will stop working (Or if CapsLock button pressed). It is better to use event.code, which does not depend on the user's chosen language.

**Example Problem**
![image](https://github.com/user-attachments/assets/5a87a936-864b-48c8-baf1-70f9de7d363c)

**Suggestion**
```suggestion
            if ((event.ctrlKey || event.metaKey) && event.code === 'KeyS') {
```

Reply:
Thanks, I'll fix soon